static/src/javascripts/projects/commercial/modules/third-party-tags/simple-reach.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import config from 'lib/config';
+
+const shouldRun =
+    !config.page.isFront &&
+    config.switches.simpleReach &&
+    config.page.isPaidContent;
+
+let simpleReachUrl = '';
+
+if (shouldRun) {
+    const authors = config.page.author.split(',');
+    const channels = config.page.sectionName.split(',');
+    const keywords = config.page.keywords.split(',');
+
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
it would be good to add a comment explaining why this rule is disabled here (unfortunately you can't lint for that, afaik...)